packages/asc-ui/src/components/Grid/Column.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Theme } from '@datapunt/asc-core'
+import ColumnStyle from './ColumnStyle'
+
+type Props = {
+  children: React.ReactNode
+  debug?: boolean
+  debugColor?: string
+  id?: keyof Theme.GridInterface
+  order?: Theme.TypeSpan
+  push?: Theme.TypeSpan
+  span: Theme.TypeSpan
+  styles?: any
+}
+
+type Context = {
+  parentSpan?: Theme.TypeSpan
+}
+
+const ParentContext = React.createContext<Context>({})
+
+const Column: React.FC<Props> = ({ children, span, id, ...props }) => {
+  const { parentSpan } = React.useContext<Context>(ParentContext)
+
+  return (
+    <ParentContext.Provider value={{ parentSpan: span }}>
```

Comment:
When is this context provider used?

Reply:
Context is necessary to allow columns to be nested. All column widths are relative and having a relative width of an element that's inside another element that already has a relative width will set a wrong value for the width unless the nested element is aware of the width of its parent.

Reply:
Ah. Couldn't find any other Consumers components or useContext hooks in the PR

Reply:
Not even on line 23? ;)

Reply:
Ah yes, but won't parentSpan always be the same as the span prop in this scenario? Or am I missing some context?

Reply:
Well, the point is that `parentSpan` (which comes from the context) will have the value of `span` which comes from the component that wraps another component.